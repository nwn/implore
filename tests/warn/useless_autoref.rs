use implore::impl_op;

#[derive(Copy, Clone, PartialEq, Eq, Debug)]
struct X;

#[impl_op(autoref)]
fn sub(lhs: X, _rhs: X) -> X {
    lhs
}

fn main() {}
