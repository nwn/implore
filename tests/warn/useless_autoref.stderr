warning: option `autoref` has no effect
 --> $DIR/no_ref_autoref.rs:6:11
  |
6 | #[impl_op(autoref)]
  |           ^^^^^^^
